ecl/eclagent/eclagent.cpp

```diff
@@ -1119,7 +1119,7 @@ void EclAgent::setResultData(const char * stepname, unsigned sequence, int len,
 
 void EclAgent::doSetResultString(type_t type, const char *name, unsigned sequence, int len, const char *val)
 {
-    LOG(MCsetresult, unknownJob, "setResultString(%s,%d,'%.*s')", nullText(name), sequence, len, val);
+    LOG(MCsetresult, unknownJob, "setResultString(%s,%d,(%d bytes))", nullText(name), sequence, len);
```

Comment:
really len should be unsigned, and this should be %u, but not significant enough to hold this up.